### ISSUE TITLE
Sorting and Faceting of Collections

### ISSUE THREAD

**Issue by JessWaggs** [UCSCLibrary/dams_project_mgmt#437]
### Descriptive Summary

As an end user, I want to be able to sort the list of collections in the DAMS both alphabetically and by newest added. As an end user, I want to be able to use facets to filter collections by criteria such as subject/topic. 

### Background 

As we continue to add collections to our DAMS, providing users with ways to sort and filter collections will improve user efficiency and effectiveness in finding relevant collections. 

### Acceptance Criteria

This is what done looks like

* [ ] Users can sort collections displayed on the A-Z collections page alphabetically
* [ ] Users can sort collections displayed on the A-Z collections page by date added
* [ ] Users can use facets to filter the collections displayed on the A-Z collections page

**Comment by JessWaggs** [comment]
@NedHenry Do we need any more detail on the sorting/faceting in this ticket?
@rschwab Do we need a mock up for adding these features to the A-Z page?

**Comment by rschwab** [comment]
@JessWaggs Do you want it styled exactly like the collection landing page? If so, I don't need a mock up. If there are minor changes we can probably talk through them.

**Comment by JessWaggs** [comment]
@rschwab Yes, I'm picturing the facets and sorting features looking like the ones on the collection landing page
